src/pages/publication/contributors_tab/ContributorValidator.tsx

```diff
@@ -1,61 +1,68 @@
-import React, { Dispatch } from 'react';
+import React from 'react';
 import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 
-import { MenuItem, TextField } from '@material-ui/core';
+import { MenuItem } from '@material-ui/core';
 import PersonIcon from '@material-ui/icons/Person';
 
-import { removeContributor, updateContributor } from '../../../redux/actions/contributorActions';
 import ContributorType from '../../../types/contributor.types';
 import StyledContributor from './StyledContributor';
+import ContributorStyles from './StyledContributor';
+import { Field } from 'formik';
 
 const StyledContributorValidator = styled(StyledContributor.ContributorContainer)`
-  grid-template-areas: 'icon name institution verify-person verify-person . delete';
+  grid-template-areas: 'icon name institution verify-person verify-person arrows delete';
 `;
 
-const StyledNameInput = styled(TextField)`
+const StyledNameInput = styled(Field)`
```

Comment:
Fikk du dette til å virke? Jeg fikk ikke til å extende Field med styled-components tidligere

Reply:
Virker for meg

Reply:
daså, nice!